### PR TITLE
Added custom video capture sample

### PR DESCRIPTION
**Pull request by rktalusani**

### CODE REVIEW

File: Custom-Video-Capturer/Custom-Video-Capturer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+		0CC86404299C7C760027D30F /* OTMTLVideoRenderer.mm */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.objcpp; path = OTMTLVideoRenderer.mm; sourceTree = "<group>"; };
+		0CC86405299C7C760027D30F /* OTMTLVideoView.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = OTMTLVideoView.m; sourceTree = "<group>"; };
+		0CC86409299C7E840027D30F /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist; path = Info.plist; sourceTree = SOURCE_ROOT; };
+		6E6B17B13FE6821CFFF00E5F /* Pods_Custom_Video_Capturer.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = Pods_Custom_Video_Capturer.framework; sourceTree = BUILT_PRODUCTS_DIR; };
+		7B1B61EF1EEF16E284F8329B /* Pods-Custom-Video-Capturer.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-Custom-Video-Capturer.debug.xcconfig"; path = "Target Support Files/Pods-Custom-Video-Capturer/Pods-Custom-Video-Capturer.debug.xcconfig"; sourceTree = "<group>"; };
```
[the remark this answers is not in the document]
Comment (rktalusani) [reply Custom-Video-Capturer/Custom-Video-Capturer.xcodeproj/project.pbxproj]: done

**Comment by jvalli** [comment]
> AVCaptureDeviceDiscoverySession warning is there in Mac SDK too so i didn't touch it.

Ok, this is something that we will need to investigate.
Did you notice any problems at runtime?

**Comment by rktalusani** [comment]
no runtime issues

**Comment by IGitGotIt** [comment]
When I close the app, the audio still goes on

**Comment by IGitGotIt** [comment]
Typically I like to add a subscribe to self flag in the code , so I don't have to go outside the app to start a session, publish etc 

```objc
void publisher_on_stream_created(otc_publisher *publisher, void *user_data, const otc_stream *stream) {
     session_on_stream_received(session, user_data, stream);
}
```
You decide if you want it.. not an issue for me..

**Comment by rktalusani** [comment]
are we ok to merge now? any blocking issues?

**Comment by jvalli** [comment]
> are we ok to merge now? any blocking issues?

I am still seeing pods configuration on the xcodeproj, did you run `pod deintegrate`?

**Comment by IGitGotIt** [comment]
> are we ok to merge now? any blocking issues?

Raj, did https://github.com/opentok/opentok-macos-sdk-samples/pull/7#issuecomment-1442599367 get fixed? Thanks

**Comment by rktalusani** [comment]
> When I close the app, the audio still goes on
fixed by implementing window close handler.

Also ran pod deintegrate

**Review by IGitGotIt — APPROVED** [review]
+1

**Review by jvalli — APPROVED** [review]
LGTM!

**Comment by v-kpheng** [comment]
Merging, since @jvalli tested already and @chetanvangadiTokbox gave us a 👍.

**Pull request merged**